source/adapters/level_zero/device.hpp

```diff
@@ -141,6 +141,17 @@ struct ur_device_handle_t_ : _ur_object {
   // Returns whether immediate command lists are used on this device.
   ImmCmdlistMode ImmCommandListUsed{};
 
+  // Returns whether large allocations are being used or not.
+  // On some Intel GPUs, this influences how kernels are compiled.
+  // If large allocations (>4GB) are requested, then kernels are
+  // compiled with stateless access.
+  // If small allocations (<4GB) are requested, then kernels are
```

Comment:
This is not accurate, even if -ze-opt-greater-than-4GB-buffer-required is not specified, kernels may still be compiled in stateless mode.

Reply:
thanks, I will remove the comment